src/test/run-pass/tcp-stress.rs

```diff
@@ -9,6 +9,7 @@
 // except according to those terms.
 
 // ignore-android needs extra network permissions
+// TODO This comment is weird as openbsd is not mentioned in the file
```

Comment:
Yeah I'm not sure what's going on here either, but it's fine to just remove the TODO